appender_test.go

```diff
@@ -86,9 +89,16 @@ func TestAppender(t *testing.T) {
 		if u64 > 9223372036854775807 {
 			u64 = 9223372036854775807
 		}
+
+		b, err := uuid.New().MarshalBinary()
```

Comment:
Since we are using `go1.18` it is not possible to do so. I get error : `conversion of slices to arrays requires go1.20 or later`
But yeah agreed once we migrate to newer versions it should be simplified as well. I thought we can upgrade the go version in a separate PR. I will raise another PR for the upgrade.